doc/CollisionSolverEntities.doc.md

```diff
@@ -0,0 +1,43 @@
+# Collision Solver for Entities other than Isaac
+
+## Label
+
+`collision`
+
+## Parameters
+
+| Label | Type | Size/Struct | Description |
+| ----- | ---- | ----------- | ----------- |
+| b | register | 1 byte | the 4 left bits are flags which indicates which side of the element the entity touched (right, left, lower, upper) |
+| hl | register | 2 bytes | address to the caracter's sheet of the element |
+| collisionSolver_.collidingEntity | fixed address | element pointer | address to the element representing the entity |
+
+## Return
+
+None
+
+## Pseudo Code
+
+~~~C
+// a function for when an entity collides with any element
+char void solve_collision_to_Isaac (
+	element entity,
+	char touched_from,
+	element.sheet* s) {

```

Comment:
Make more clear the link between the function parameters (like entity) and those given in the Parameter section.